src/net/cruhland/axioms/CoreAlgebra/Group.agda

```diff
@@ -0,0 +1,43 @@
+open import net.cruhland.axioms.CoreAlgebra.Monoid using (Monoid)
+
+module net.cruhland.axioms.CoreAlgebra.Group (G : Set) (M : Monoid G) where
```

Comment:
It'll be more convenient to put these parameters on the `Group` record instead:

```agda
record Group (G : Set) {{_ : Eq G}} {{_ : Monoid G}} : Set₁ where
```

You may start needing to move other fields, like `_⊙_` and `identity` into the parameters for `Monoid` as well, if there are multiple instances of `Monoid` for a type `G`. Otherwise Agda won't be able to determine a unique instance.

Alternatively, if instances start to become a pain, you could make `Monoid` a normal parameter so that it must always be explicitly given.